### PR TITLE
Responsive YouTube video

### PR DESCRIPTION
**Pull request by kubami9**
Added:
-ytVideo id in index.html to manipulate video
-media queries so now video looks good on mobile devices and isn't cut in a half
I also included some comments so other contributors can easily follow with no headache :)

### CODE REVIEW

**Comment by kubami9** [comment]
Here is the preview link, so you can check it out how it will look now:
https://kubami9.github.io/kubamichalski-gci-contributing/

**Review by shubham-padia — COMMENTED** [review]
The video now only covers half of the width on the desktop version. The previous version looked good on desktop, please make changes so that the video also looks good on desktop

**Comment by kubami9** [comment]
Okay, I'll solve it

**Comment by abishekvashok** [comment]
conflicts

**Review by abishekvashok — COMMENTED** [review]
Use col of bootstrap to make it responsive.

**Comment by kubami9** [comment]
Woooow, nearly whole website is in conflict, it looks like it has been rebuilt

**Comment by kubami9** [comment]
Okay, never mind, made some mistake, already fixed it 😄

**Comment by kubami9** [comment]
Please check it now, it should be good

File: css/custom.css [hunk 1]
```diff
@@ -54,3 +54,30 @@
     padding: 7px;
     box-sizing: content-box;
 }
+
+/* Media Queries for YouTube Video */
+/* ytVideo is ID for video */
+@media (min-width: 992px) {
+    #ytVideo {
+        width: 880px;
```
Comment (abishekvashok) [inline css/custom.css]: Don't use media queries and static values, use bootstraps responsive classes like s12

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: css/custom.css (hunk 1 above)
Comment (kubami9) [reply css/custom.css]: Ohh okay, sorry I'll fix it

**Comment by kubami9** [comment]
Done 😃

File: index.html [hunk 2]
```diff
@@ -586,11 +589,10 @@ <h3 class="mb40 mb-xs-24">FOSSASIA Google Code-In 2016/17</h3>
 		                    </p>
 		                </div>
 		            </div>
-
 <div class="widget video-container">
-				<iframe width="600" height="400" src="https://www.youtube.com/embed/SLl--BrmAXs?list=PLzZVLecTsGpK544UhC1_Gm4sl4eW3lTEq" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>
+				<iframe class="col-xs-12 col-sm-12 col-md-12 col-lg-10" src="https://www.youtube.com/embed/SLl--BrmAXs?list=PLzZVLecTsGpK544UhC1_Gm4sl4eW3lTEq" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>					
 				</div>		        </div>
-
+
```
Comment (anshumanv) [inline index.html]: Please remove these empty spaces. Rest all looks good! :smile:

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by kubami9** [comment]
Done 👍

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Please squash your commits from next time.
Great Work!

Cheers.

**Pull request merged**